File: helfomat-api/src/main/java/de/helfenkannjeder/helfomat/api/organization/OrganizationNotFoundException.java
package de.helfenkannjeder.helfomat.api.organization;

import de.helfenkannjeder.helfomat.core.organization.OrganizationId;

/**
 * @author Valentin Zickner
 */
public class OrganizationNotFoundException extends RuntimeException {
    public OrganizationNotFoundException(OrganizationId organizationId) {
        super("Organization with '" + organizationId.getValue() + "' could not be found.");
    }
}

File: helfomat-api/src/main/kotlin/de/helfenkannjeder/helfomat/api/organization/OrganizationNotFoundException.kt
package de.helfenkannjeder.helfomat.api.organization

import de.helfenkannjeder.helfomat.core.organization.OrganizationId

/**
 * @author Valentin Zickner
 */
class OrganizationNotFoundException(organizationId: OrganizationId) : RuntimeException("Organization with '" + organizationId.value + "' could not be found.") 